### PR TITLE
Expanded Seeded data & New Customer Form

### PR DESCRIPTION
**Pull request by csmalley89**
## Pull Request Description

Expanded the seeded data to reflect ProductTypes and SubProductTypes that would closer reflect that of an ecommerce site such as Amazon.com.

Built cshtml form to add customers to the bangazon site. 
## Steps to Test
1. When the customer adds a product to Bangazon, the dropdown menu will have a much larger selection of SubProductTypes to choose from.
2. Need customer controller to be finished in order to test New Customer form
## Link to Feature Ticket

 #89 
#43

### CODE REVIEW

**Review by Jgregnier — COMMENTED** [review]
Looks good to me Except for the new.cshtml file. Don't know enough about forms to critique it.

File: Data/DBInitializer.cs [hunk 1]
```diff
@@ -62,20 +65,95 @@ public static void Initialize(IServiceProvider serviceProvider)
...
                   new SubProductType { 
-                      Name = "La Croix",
-                      Description = "Cool La Croix taste really good",
-                      ProductTypeId = productTypes.Single(s => s.Name == "Food and Drink").ProductTypeId
+                      Name = "Ranges, Ovens, and Components",
+                      Description = "Appliances to heat food",
+                      ProductTypeId = productTypes.Single(s => s.Name == "Appliances").ProductTypeId
+                  },
+                  new SubProductType { 
+                      Name = "Small Applicances",
```
Comment (delaineackerman) [inline Data/DBInitializer.cs]: Small Appliances*

File: Data/DBInitializer.cs [hunk 2]
```diff
@@ -62,20 +65,95 @@ public static void Initialize(IServiceProvider serviceProvider)
...
+                      Description = "A varity of small household appliances from coffee makers to irons to desk fans",
```
Comment (delaineackerman) [inline Data/DBInitializer.cs:84]: A variety*

File: Data/DBInitializer.cs [hunk 3]
```diff
@@ -62,20 +65,95 @@ public static void Initialize(IServiceProvider serviceProvider)
...
+                      ProductTypeId = productTypes.Single(s => s.Name == "Appliances").ProductTypeId
                   },
                   new SubProductType { 
-                      Name = "Kitchen",
-                      Description = "Appliances for your kitchen",
+                      Name = "Washing Machine, Dryers, and Dishwashers",
+                      Description = "Appliances to clean",
                       ProductTypeId = productTypes.Single(s => s.Name == "Appliances").ProductTypeId
                   },
+                  new SubProductType {
```
Comment (delaineackerman) [inline Data/DBInitializer.cs:92]: We already have Microwaves up above

File: Data/DBInitializer.cs [hunk 4]
```diff
@@ -62,20 +65,95 @@ public static void Initialize(IServiceProvider serviceProvider)
...
+                      ProductTypeId = productTypes.Single(s => s.Name == "Electronics and Media").ProductTypeId
+                  },
+                  new SubProductType { 
+                      Name = "Video Games and Gaming Hardware",
+                      Description = "Gaming Consoles, Video Games, and Components",
+                      ProductTypeId = productTypes.Single(s => s.Name == "Electronics and Media").ProductTypeId
+                  },
+                  new SubProductType { 
+                      Name = "Home Theatre and Audio",
+                      Description = "Everything from TVs, Speakers, Bluray Players, and all the Components.",
```
Comment (delaineackerman) [inline Data/DBInitializer.cs:109]: Blu-ray*

File: Data/DBInitializer.cs [hunk 5]
```diff
@@ -62,20 +65,95 @@ public static void Initialize(IServiceProvider serviceProvider)
...
+                      ProductTypeId = productTypes.Single(s => s.Name == "Beauty, Health, and Grocery").ProductTypeId
+                  },
+                  new SubProductType { 
+                      Name = "Furniture",
+                      Description = "From kitchen tables to sofas to mattresses",
+                      ProductTypeId = productTypes.Single(s => s.Name == "Everything Else").ProductTypeId
+                  },
+                  new SubProductType { 
+                      Name = "Clothing",
+                      Description = "If you can wear it and its made of cloth, you will find here. Yeah we know technically you can wear almost anything but please keep it with stuff you might actually find at a clothing store. ",
```
Comment (delaineackerman) [inline Data/DBInitializer.cs:149]: and it's*

**Review by thrashMoneyVillainare — COMMENTED** [review]
LGTM

**Review by delaineackerman — COMMENTED** [review]
See Comments

**Comment by HubDub** [comment]
looks good

**Review by JamsDuke — APPROVED** [review]
Looks like youll be good after implementing Delaine's changes.

**Pull request merged**